packages/modules/src/modules/banners/common/choiceCard/ChoiceCardInteractive.tsx

```diff
@@ -116,6 +147,30 @@ const buildStyles = (design: ChoiceCardSettings | undefined, frequencyColumns: n
             ${until.mobileMedium} {
                 font-size: 10px;
             }
+
+            & + label {
+                ${buttonTextColour ? `color: ${buttonTextColour};` : ''}
```

Comment:
Would it simplify/slightly shorten things to use `&&` instead of the ternary? It would allow the default empty string at the end to be dropped in all cases I think.

```
${buttonTextColour && `color: ${buttonTextColour};`}
```

Reply:
Agreed, and done.